easybuild/easyconfigs/q/Qhull/Qhull-2015.2-intel-2016a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'Qhull'
+version = '2015.2'
+
+homepage = 'http://www.qhull.org'
+description = """
+Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point,
+furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d,
+and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+#recently checkged the Unix version numbering see https://github.com/Homebrew/homebrew-science/issues/3353
```

Comment:
point to http://www.qhull.org/src/Changes.txt instead?

also, use space after `#` (our automatic style checking thing we're working on will trip over not doing so)